src/compiler/scala/tools/nsc/tasty/bridge/ContextOps.scala

```diff
@@ -189,8 +191,19 @@ trait ContextOps { self: TastyUniverse =>
 
     final def newLocalDummy: Symbol = owner.newLocalDummy(u.NoPosition)
 
-    final def newWildcardSym(info: Type): Symbol =
-      owner.newTypeParameter(u.nme.WILDCARD.toTypeName, u.NoPosition, u.NoFlags).setInfo(info)
+    final def newWildcard(info: Type): Symbol =
+      owner.newTypeParameter(
+        name     = u.freshTypeName("_$")(u.currentFreshNameCreator),
```

Comment:
is this correct?

Reply:
The difficulty with fresh names is that they should be deterministic to ensure that the compiler is stable, also under inremental compilation (recompile a subset of sources of a project). For code being compiled, there's a fresh-name-creator in each compilation unit. Since type checking happens lazily it's a bit more tricky there. The Scala 2 unpickler doesn't need to don any fresh names it seems, so we can't compare with that. But if the names created here don't leak out in any way to code being compiled, i think using the global fresh name creator should be OK.

cc @retronym

Reply:
calls to `newWildcard` will only be made to provide quantifiers for existential types, (which must be made because tasty encodes wildcards as in-place TypeBounds). I assume existential quantifiers can't be referenced  elsewhere